miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -158,11 +155,8 @@ abstract class MiniAppMessageBridge {
      * Set implemented userInfoBridgeDispatcher.
      * Can use the default provided class from sdk [UserInfoBridgeDispatcher].
      **/
-    fun setUserInfoBridgeDispatcher(bridgeDispatcher: UserInfoBridgeDispatcher) {
-        userInfoBridgeDispatcher = bridgeDispatcher
-        if (miniAppViewInitialized)
-            userInfoBridgeDispatcher.init(bridgeExecutor, customPermissionCache, miniAppId)
-    }
+    fun setUserInfoBridgeDispatcher(bridgeDispatcher: UserInfoBridgeDispatcher) =
```

Comment:
From now, if HostApp forget to invoke this function, then I think it will cause issue. 

e.g. in sample app, I was noticing unlimited loading when not calling this function.
![Screen Shot 2021-01-22 at 15 19 07](https://user-images.githubusercontent.com/66930373/105455122-57211d00-5cc6-11eb-973e-223e00565fec.png)

Reply:
I noticed that in the master branch implementation, if user does not invoke this function, there is no response or show loading.
The error should be thrown so I will throw the error then.

Reply:
I have updated this part. You will see:

In master branch, there is no response in js sample mini app (user details screen did not handle ui here but Auth screen does so you can check it) and no error sent to miniapp.
In this refactor, there is also no response but the error has been sent to miniapp, developer can see it in chrome log.

Reply:
Thank you for this update, I think this is a good improvement.